### PR TITLE
Fix-88 Starts building image-inspector image at registry.centos.org

### LINKED ISSUE #88: Start building image-inspector container image at CentOS community container pipeline
This issue is to track indexing image-inspector repository to build at [CentOS community container pipeline service](https://github.com/CentOS/container-pipeline-service).

A successful build indexes image at https://registry.centos.org/

Docker pull URL like registry.centos.org/namespace/image-inspector

Since `centos:7` is used as a base image, the Dockerfile should not need many changes to onboard project on build service.

With project configured to build at pipeline service, there will be automatic rebuilds of image, when
  - there is a change pushed to the repository
  - the underlying base image got an update

To index this project to build at pipeline service, all it needs is to add a YAML file `cccp.yml` in the project repository with 
```
job-id: image-inspector
```

### PR DESCRIPTION
**Pull request by navidshaikh**
Fixes #88

  Adds a file cccp.yml with `job-id: image-inspect`, needed to build
  at registry.centos.org

### CODE REVIEW

**Comment by enoodle** [comment]
@navidshaikh Thanks,

Can you later add this to https://github.com/CentOS/container-index ?

**Comment by simon3z** [comment]
@enoodle can you review?

**Comment by navidshaikh** [comment]
> Can you later add this to https://github.com/CentOS/container-index ?

Raised a PR  https://github.com/CentOS/container-index/pull/250 , what should be the preferable email to receive build notifications updates on?

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM centos:7
+FROM registry.centos.org/centos/centos:latest
```
Comment (enoodle) [inline Dockerfile]: Why use `latest` instead of `7` tag?

**Comment by enoodle** [comment]
> what should be the preferable email to receive build notifications updates on?

You can put @simon3z 's if he wants it there or otherwise mine.

**Comment by navidshaikh** [comment]
> Why use latest instead of 7 tag?

Updated the tag to `7`, base image is now `registry.centos.org/centos/centos:7`

> You can put @simon3z 's if he wants it there or otherwise mine.

Updated https://github.com/CentOS/container-index/pull/250 with @enoodle email for now, I will update as @simon3z confirms.

**Pull request merged**